src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -41,43 +40,40 @@
   private StashPluginConfiguration config;
   private File projectBaseDir;
   private final InputFileCache inputFileCache;
-  private CoverageProjectStore coverageProjectStore;
 
   public StashRequestFacade(StashPluginConfiguration stashPluginConfiguration,
-                            InputFileCache inputFileCache,
-                            StashProjectBuilder projectBuilder,
-                            CoverageProjectStore coverageProjectStore) {
+      InputFileCache inputFileCache,
+      StashProjectBuilder projectBuilder) {
     this.config = stashPluginConfiguration;
     this.inputFileCache = inputFileCache;
     this.projectBaseDir = projectBuilder.getProjectBaseDir();
-    this.coverageProjectStore = coverageProjectStore;
   }
 
-  public List<Issue> extractIssueReport(ProjectIssues projectIssues) {
-    return SonarQubeCollector.extractIssueReport(projectIssues, this);
+  public List<Issue> extractIssueReport(ProjectIssues projectIssues, Project project) {
+    return SonarQubeCollector.extractIssueReport(projectIssues, this, project);
+  }
+
+  private MarkdownPrinter getMarkdownPrinter() throws StashConfigurationException {
+    return new MarkdownPrinter(config.getStashURL(), getPullRequest(), this, getIssueThreshold(),
+        config.getSonarQubeURL());
   }
 
   /**
    * Post SQ analysis overview on Stash
    */
   public void postAnalysisOverview(PullRequestRef pr,
-                                   int issueThreshold,
-                                   List<Issue> issueReport,
-                                   StashClient stashClient) {
+      int issueThreshold,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "issueThreshold". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)